Assets/Project/Commons/BossPhaseStateMachine/Scripts/Presenter/BossPhaseState.cs

```diff
@@ -0,0 +1,20 @@
+using Project.Commons.BossPhaseStateMachine.Scripts.Model;
+
+namespace Project.Commons.BossPhaseStateMachine.Scripts.Presenter
+{
+    public abstract class BossPhaseState
+    {
+        protected BossHealthModel HealthModel;
+        protected BossPhaseStateMachine StateMachine;
```

Comment:
変数系はキャメルケースに統一したい気持ちがある。
getter, setter系をパスカルケースにするため

```suggestion
        protected BossHealthModel healthModel;
        protected BossPhaseStateMachine stateMachine;
```